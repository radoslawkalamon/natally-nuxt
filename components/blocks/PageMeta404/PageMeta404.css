.page-meta-404 {
  text-align: center;
}
