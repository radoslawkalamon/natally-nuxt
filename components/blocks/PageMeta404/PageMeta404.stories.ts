import type { Meta, Story } from '@storybook/vue'
import BlocksPageMeta404 from '@/components/blocks/PageMeta404/PageMeta404.vue'

const meta: Meta = {
  title: 'Blocks / Page Meta 404',
  component: BlocksPageMeta404
}
export default meta

export const Default: Story = () => ({
  components: {
    BlocksPageMeta404
  },
  template: '<BlocksPageMeta404 />'
})
