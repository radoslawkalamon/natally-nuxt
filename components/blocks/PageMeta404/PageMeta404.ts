import Vue from 'vue'

export default Vue.extend({
  name: 'BlocksPageMeta404',
  components: {
    ComponentsLink: () => import(/* webpackChunkName: "components" */'@/components/components/Link/Link.vue'),
    ComponentsTitle: () => import(/* webpackChunkName: "components" */'@/components/components/Title/Title.vue'),
    WrappersText: () => import(/* webpackChunkName: "wrappers" */'@/components/wrappers/Text/Text.vue'),
    WrappersSection: () => import(/* webpackChunkName: "wrappers" */'@/components/wrappers/Section/Section.vue')
  }
})
