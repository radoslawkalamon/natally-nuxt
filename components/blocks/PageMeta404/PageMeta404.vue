<template>
  <WrappersSection
    class="page-meta-404"
    data-test="blocks-page-meta-404"
  >
    <template #header>
      <ComponentsTitle title="Porobiło się!" />
    </template>
    <WrappersText>
      <p>Strona, której szukasz nie istnieje lub została usunięta!</p>
    </WrappersText>
    <template #footer>
      <ComponentsButtonLink
        label="Wróć na Stronę Główną"
        to="/"
      />
    </template>
  </WrappersSection>
</template>

<script src="./PageMeta404.ts" lang="ts"></script>
<style src="./PageMeta404.css" scoped></style>
