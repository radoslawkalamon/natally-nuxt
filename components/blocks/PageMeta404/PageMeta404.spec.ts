import PageMeta404 from '@/components/blocks/PageMeta404/PageMeta404.vue'
import { shallPassIntegrationSanityTest, shallPassUnitSanityTest } from '@/devtools/jest.common.spec'

describe('Blocks / Page Meta 404', () => {
  describe('Unit', () => {
    shallPassUnitSanityTest({
      component: PageMeta404,
      options: {
        stubs: [
          'ComponentsButtonLink',
          'ComponentsTitle',
          'WrappersText',
          'WrappersSection'
        ]
      }
    })
  })

  describe('Integration', () => {
    shallPassIntegrationSanityTest({
      component: PageMeta404
    })
  })
})
