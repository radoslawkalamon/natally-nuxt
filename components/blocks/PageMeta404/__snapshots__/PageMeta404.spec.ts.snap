// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blocks / Page Meta 404 Integration shall pass integration sanity test 1`] = `
"<section class="section page-meta-404">
  <div data-test="wrappers-section-header" class="section__header">
    <h1 class="title title--design-1 title--underscore">Porobiło się!</h1>
  </div>
  <div class="section__inner">
    <div class="text">
      <p>Strona, której szukasz nie istnieje lub została usunięta!</p>
    </div>
  </div>
  <div data-test="wrappers-section-footer" class="section__footer">
    <nuxtlink to="/" class="link link--button">Wróć na Stronę Główną</nuxtlink>
  </div>
</section>"
`;

exports[`Blocks / Page Meta 404 Unit shall pass unit sanity test 1`] = `
"<wrapperssection-stub class="page-meta-404">
  <wrapperstext-stub>
    <p>Strona, której szukasz nie istnieje lub została usunięta!</p>
  </wrapperstext-stub>
</wrapperssection-stub>"
`;
